test/restart/mzcompose.py

```diff
@@ -126,11 +131,6 @@ def workflow_github_17578(c: Composition) -> None:
               WHERE top_level_s.name = 'with_subsources' AND (s.type = 'progress' OR s.type = 'subsource');
             source          subsource
             -------------------------
-            with_subsources accounts
-            with_subsources auctions
-            with_subsources bids
-            with_subsources organizations
-            with_subsources users
```

Comment:
I'm not sure what the intent of this validation is, but we could rework this query to retrieve the tables from `mz_tables` too since we have each id from the `mz_object_dependencies` result

Reply:
Addressed with https://github.com/MaterializeInc/materialize/compare/d05c2c8220a167fdfa57eb8f66d4c8c534abd4ef..ef8b394a6a26d8c3ac294aefe16a041b1b43fa91.